### DISCUSSION TITLE
How to add a remote to a new local repo?

### DISCUSSION THREAD

**Discussion by benfrain** [jesseduffield/lazygit#2041] [Q&A] [answered]
New user. I have a local repo set up with Lazygit. I have a blank project setup on GitLab and want to add it as the remote with this kind of command:

`git remote add origin blah-blah:benfrain/mini-guide.git`

Is there a way to do that in Lazygit?

**Comment by mark2185** [comment, marked as answer]
Sure there is!

Go to the branches panel (`3`), press `]` to enter the `Remotes` view and check the menu with available options by pressing `x`, but I think it's under `n`.

**Reply by benfrain** [reply]
Thanks, I had totally missed that there was 'tabs' in each section!

**Reply by mark2185** [reply]
No problem, glad to have helped! :)

Feel free to watch [15 lazygit features in under 15 minutes](https://www.youtube.com/watch?v=CPLdltN7wgE) which should show off the more common `git` actions through `lazygit`.
